src/components/Login/Login.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+export class Login extends React.Component{
```

Comment:
Your JSX is a mess ... clean it up. Also where are all those css classes from? The whole thing looks copy-pasted